filesystems/samba2/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+name: samba2
+services:
+  test-samba-share:
+    volumes:
+      - ./smb.conf:/etc/samba/smb.conf
+    ports:
+      - 139:139
+      - 137:137/udp
+      - 138:138/udp
+      - 445:445
+    image: scitotec/test-samba-share
+    command: -p -u 'foo;badpass' -u 'bar;badpass' 
```

Comment:
We could use this command to the docker image. And isn't it nicer to have an testiaf user?

Reply:
Yes, I'll change it to the users we use in the unit test. That's frankframework user.